CITATION.cff

```diff
@@ -0,0 +1,32 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: "GLASS: Generator for Large Scale Structure"
+message: >-
+  If you use this software, please cite it using the metadata from this file.
+type: software
+authors:
+  - given-names: Nicolas
+    family-names: Tessore
+  - given-names: Arthur
+    family-names: Loureiro
+  - given-names: Benjamin
+    family-names: Joachimi
+  - given-names: Maximilian
+    family-names: von Wietersheim-Kramsta
+  - given-names: Niall
+    family-names: Jeffrey
+identifiers:
+  - type: doi
+    value: 10.21105/astro.2302.01942
+  - type: other
+    value: "https://ui.adsabs.harvard.edu/abs/2023OJAp....6E..11T"
+    description: Provided by the SAO/NASA Astrophysics Data System
```

Comment:
I'm not sure what - other than advertising for ADS itself - this achieves. The DOI will link everything.